### ISSUE TITLE
Table Markup in Headers

### ISSUE THREAD

**Issue by marcdexet-cnrs** [asciidoctor/asciidoctor-pdf#1075]
Hi,

## The need

I still need to include this kind of _table in header_ for my very formatted documentation.

![](https://pbs.twimg.com/media/DQ82z1KX0AAkqlZ.jpg)

## The actual work
I'm successful to produces a near perfect one 

![image](https://user-images.githubusercontent.com/4507672/58315069-15fda900-7e11-11e9-8e1d-a410ff2a20a1.png)

## The situation

@mojavelinux has written in the pas (3 years ago, an epoch) in the old and closed issue #393,

> I'm not going to say never, because as we build better abstractions, harder things become possible. But right now that would be too difficult to implement and lower priority than some really pressing issues we've got still floating.

It is possible now or is it still a far away dream ?

Maybe just a *cell* capability could be sufficient , just 3 bordered cells as a 

    <div style="border: 1px solid black;">

As a lot of related issues could tell it, it's a very pretty feature for a lot of us.

Thanks for the job.

**Comment by mojavelinux** [comment]
Someone could come along with solution that proves me wrong, but I don't think it's feasible with Asciidoctor PDF. We're very near the limit of what we can do with this builder (without introducing an unmaintainable amount of complexity).

That's exactly why we're starting to explore a new solution that uses the browser and web standards to layout the pages (using a headless browser driver like puppeteer). Once we make that switch, then you'll be able to put content where ever you like without having to wait on incremental improvements to the layout  engine.

**Comment by mojavelinux** [comment]
I could maybe see allowing borders on each of the columns in the running content. That wouldn't be unreasonable.

**Comment by marcdexet-cnrs** [comment]
It would be a very affordable solution from my point of view for everybody :)

**Comment by mgenereu** [comment]
I'd like to help with the workaround or the new engine for the same reason.

**Comment by marcdexet-cnrs** [comment]
Hi @mojavelinux and @mgenereu. How to make this little feature "Allowing borders on each of the columns in the running content" put on the roadmap ?

By creating a new issue ?

**Comment by mojavelinux** [comment]
Yes, please. This issue is too general and will easily get missed. A dedicated issue will be easier to schedule.
